spot_examples/spot_examples/batch_trajectory.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+"""
+This example illustrates the methodology for dispatching extended-duration
```

Comment:
Can you add to the README of `spot_examples` to mention that this example has been added?

Reply:
Added.

Reply:
Thanks! Do we also need to add it to `spot_examples/setup.py`, or is it meant to be run just via python?

Reply:
Good catch. Added.